File: LB-Assignments/Assignment_3/program1.c
#include<stdio.h>

void PrintEven(int iNo)
{
    if(iNo <= 0)
    {
        return;
    }

    int iCnt = 1;
    int iFrequency = 1;

    while(iFrequency<= iNo)
    {
        if(iCnt % 2 == 0)
        {
            printf("%d  ",iCnt);
            iFrequency++;
        }
        iCnt++;
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    PrintEven(iValue);

    return 0;
}

File: LB-Assignments/Assignment_22/program1.java
/*
Input : 5
Output : A  B  C  D  E   
*/
import java.util.*;

class Logic
{
    int iNo = 0;

    public Logic(int a)
    {
        this.iNo = a;
    }

    void Pattern()
    {
        int iCnt = 0;
        char ch = '\0';

        for(iCnt = 1, ch = 'A'; iCnt <= iNo; iCnt++, ch++)
        {
            System.out.print(ch+"\t");
        }
        System.out.println();
    }
}
class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue =0;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Logic lobj = new Logic(iValue);

        lobj.Pattern();

    }
}